### DISCUSSION TITLE
RTMP Server

### DISCUSSION THREAD

**Discussion by Bombazok** [pedroSG94/RootEncoder#1378] [Q&A] [answered]
Hi, Pedro. You have an RTSP-Server project that launches the RTSP-Server at the start of the stream. How to do the same with RTMP only (Run RTMP-Server at the start of the stream)? Perhaps there are ready-made solutions or any hints?

**Comment by pedroSG94** [comment, marked as answer]
Hello,

I haven't a project for RTMP as server and I don't know a project that work like that.
Maybe, you can develop a project like that and use my rtmp module as base to do it. You have all rtmp packets needed for it in the rtmp module
